ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__automated-release.yaml

```diff
@@ -0,0 +1,606 @@
+  cron: 28 3 16 2 *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      E2E_RUN_TAGS: '@level0'
+      EXTRACT_MANIFEST_INCLUDED: "true"
+      TEST_FILTERS: ~ChkUpgrade&;~NonPreRelease&;~Serial&;~Disruptive&;~ConnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&
+      TEST_IMPORTANCE: LEVEL0
+    test:
+    - chain: openshift-e2e-test-qe-automated-release
+    workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private-workers-rhcos-rhel8
+- as: aws-ipi-localzone-byo-subnet-ovn-f360
+  cron: 31 11 16 2 *
+  steps:
+    cluster_profile: aws-1-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      E2E_RUN_TAGS: '@level0'
+      FEATURE_GATES: '[InstallAlternateInfrastructureAWS=true]'
+      FEATURE_SET: CustomNoUpgrade
+      TEST_IMPORTANCE: LEVEL0
+    test:
+    - chain: openshift-e2e-test-qe-automated-release
+    workflow: cucushift-installer-rehearse-aws-ipi-edge-zone-byo-subnet-ovn
+- as: aws-ipi-private-shared-vpc-phz-sts-f360
+  cron: 41 18 16 2 *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      E2E_RUN_TAGS: '@level0'
+      EXTRACT_MANIFEST_INCLUDED: "true"
+      TEST_IMPORTANCE: LEVEL0
+    test:
+    - chain: openshift-e2e-test-qe-automated-release
+    workflow: cucushift-installer-rehearse-aws-ipi-private-shared-vpc-phz-sts
+- as: aws-ipi-proxy-sdn-workers-rhel8-fips-f360
+  cron: 2 15 16 2 *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      E2E_RUN_TAGS: '@level0'
+      OVERRIDE_OPENSHIFT_SDN_DEPRECATION: "true"
+      TEST_IMPORTANCE: LEVEL0
+    test:
+    - chain: openshift-e2e-test-qe-automated-release
+    workflow: cucushift-installer-rehearse-aws-ipi-proxy-sdn-workers-rhel8-fips
+- as: aws-sc2s-ipi-disc-priv-fips-f360
+  cron: 22 4 16 2 *
+  steps:
+    cluster_profile: aws-sc2s-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      E2E_RUN_TAGS: '@level0'
+      FIPS_ENABLED: "true"
+      TEST_FILTERS: ~ChkUpgrade&;~NonPreRelease&;~Serial&;~Disruptive&;~ConnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&
+      TEST_IMPORTANCE: LEVEL0
+    test:
+    - chain: openshift-e2e-test-qe-automated-release
+    workflow: cucushift-installer-rehearse-aws-sc2s-ipi-disconnected-private
+- as: aws-usgov-ipi-private-ep-fips-f360
+  cron: 9 21 16 2 *
+  steps:
+    cluster_profile: aws-usgov-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      E2E_RUN_TAGS: '@level0'
+      FIPS_ENABLED: "true"
+      TEST_IMPORTANCE: LEVEL0
+    test:
+    - chain: openshift-e2e-test-qe-automated-release
+    workflow: cucushift-installer-rehearse-aws-usgov-ipi-private-with-ep-s3
+- as: azure-ipi-confidential-trustedlaunch-f360
+  cron: 34 18 16 2 *
+  steps:
+    cluster_profile: azure-qe
+    env:
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      E2E_RUN_TAGS: '@level0'
+      TEST_IMPORTANCE: LEVEL0
```

Comment:
@liangxia do we need TEST_FILTERS to exclude some cases like L#136

Reply:
L136 is a disconnected job, so we have to override TEST_FILTERS, because default value of TEST_FILTERS contains `~DisconnectedOnly&`.

Reply:
I just wonder if disruptive cases are included e.g. MCO level0 cases, job duration will be much longer than before

Reply:
I think this has been raise and talked when adding the tags. @wsun1 has more details for this.

Anyway, we can reconsider our decision after we run the real tests for some time and get better understanding for the tests selected.